src/functions.php

```diff
@@ -133,13 +138,12 @@ function uploadImages(array $vcards, $config, $configPhonebook, callable $callba
     }
     ftp_close($ftp_conn);
 
-    error_log(PHP_EOL);
     if ($countAllImages > MAX_IMAGE_COUNT) {
-        error_log("WARNING: You have ".$countAllImages." contact images on FritzBox");
-        error_log("         FritzFon may handle only up to ".MAX_IMAGE_COUNT." images.");
-        error_log("         (see: https://github.com/andig/carddav2fb/issues/92)");
-        error_log("         Some images may not display properly.");
-        error_log("");
+        error_log(PHP_EOL."WARNING: You have ".$countAllImages." contact images on FritzBox");
+        error_log(PHP_EOL."         FritzFon may handle only up to ".MAX_IMAGE_COUNT." images.");
+        error_log(PHP_EOL."         (see: https://github.com/andig/carddav2fb/issues/92)");
+        error_log(PHP_EOL."         Some images may not display properly.");
+        error_log(PHP_EOL."");
```

Comment:
the ."" is not needed.

Reply:
Also- please simple change this to HEREDOC syntax, so much easier to read.

Reply:
I already found these multiple lines for "error_log" in your version, I just added the "EOL"s. But of course it can be changed to HEREDOC syntax, I find it much easier to read, too.